### PR TITLE
Added sort deadline function

### PR DESCRIPTION
**Pull request by Rijul24**
Changes made:
- Added a _sort deadline_ function that sorts the deadlines and then sends it out
- made a minor bug fix in `/bot.py`

### CODE REVIEW

File: dbinteract.py [hunk 1]
```diff
@@ -101,6 +102,24 @@ def fetch_timezone(guild_id):
...
+            if ans[j][1] < ans[i][1]:
+                ans[i] = list(ans[i])
+                ans[j] = list(ans[j])
+                
+                temp = ans[j]
+                ans[j] = ans[i]
+                ans[i] = temp
+                
+                ans[i] = tuple(ans[i]) 
+                ans[j] = tuple(ans[j])
```
Comment (armaanbadhan) [inline dbinteract.py]: this works fine but converting to list just to convert back to tuple is redundant. please remove that

**Comment by armaanbadhan** [comment]
please revert the changes made in bot.py, make sure to pull before pushing

File: dbinteract.py (hunk 1 above)
Comment (Rijul24) [reply dbinteract.py]: we have to convert to list to change the elements. Can't assign the elements in a tuple.
The workaround is changing the function to return list everywhere

**Comment by Rijul24** [comment]
> please revert the changes made in bot.py, make sure to pull before pushing

that was a bug fix, that line got removed while deleting the last pull request.
Without it the code doesn't work.

**Comment by Rijul24** [comment]
helo pls check

**Pull request merged**